prov/efa/src/efa_base_ep.c

```diff
@@ -185,6 +185,10 @@ int efa_qp_create(struct efa_qp **qp, struct ibv_qp_init_attr_ex *init_attr_ex)
 			efa_attr.flags |= EFADV_QP_FLAGS_UNSOLICITED_WRITE_RECV;
 #endif
 		efa_attr.driver_qp_type = EFADV_QP_DRIVER_TYPE_SRD;
+#if HAVE_EFADV_SL
+		if (tclass == FI_TC_LOW_LATENCY)
+			efa_attr.sl = tclass;
```

Comment:
This is not correct, `tclass` is not same with sl. @mrgolin suggested using 8 as a start point.

Reply:
Do they define a name for it? I want to avoid using magic number.

Reply:
It's a good callout.... I don't think there are defined macro for it. can we have that? @mrgolin

Reply:
If we are not going to have a macro from rdma-core. We can start from defining a macro inside Libfabric and reference it here.